credentials.py

```diff
@@ -58,9 +58,8 @@ def verify_jwt(credentials: Credentials) -> bool:
 
 
 async def get_token_from_bearer(
-    request: Request,
     http_credentials: HTTPAuthorizationCredentials = Depends(bearer_scheme),
-) -> Credentials:
+) -> str:
```

Comment:
I think we'd have to either specify an `Optional[str]`, or perhaps more correctly; return an empty string in the case when `http_credentials` does not resolve.